core/src/components/description-list/description-list.stories.js

```diff
@@ -0,0 +1,103 @@
+export default {
+  title: "Components/Description List",
+  argTypes: {
+    variant: {
+      options: ["regular", "compact"],
+      control: { type: "radio" },
+    },
+  },
+};
+
+const html = String.raw;
+
+const DL_configuration = {
+  regular: {
+    gridStyles: "row-gap: 1rem; column-gap: 2rem",
+    colon: "",
+    parenthesis_open: "",
+    parenthesis_closed: "",
+    comma: "",
+  },
+  compact: {
+    gridStyles: "row-gap: 0; column-gap: 1.5rem",
+    colon: html`<span class="fudis-dl-item-term__compact__colon"
+      >&colon;</span
+    >`,
+    parenthesis_open: html`<span
+      class="fudis-dl-item-details__compact__parenthesis"
+      >&#40;</span
+    >`,
+    parenthesis_closed: html`<span
+      class="fudis-dl-item-details__compact__parenthesis"
+      >&#41;</span
+    >`,
+    comma: html`<span class="fudis-dl-item-details__compact__comma"
```

Comment:
There is slight difference in space after commas (Ngx vs. Core examples), when there are multiple dd's listed. Not sure why this happens but the current Ngx version is better so is this going to break our current VR tests etc. in Ngx?

<img width="371" height="126" alt="Screenshot 2025-09-09 at 14 07 20" src="https://github.com/user-attachments/assets/af8441e2-edc5-46fc-ba49-b741a5bb5f4f" />
<img width="462" height="96" alt="Screenshot 2025-09-09 at 14 03 42" src="https://github.com/user-attachments/assets/e6cff371-dcac-48d2-9bb4-e080c0b6d2ad" />